renku/cli/run.py

```diff
@@ -25,35 +25,63 @@
 
```

Comment:
For me, the wording "previous arguments" is confusing. I would suggest something like:
```suggestion
Tracking execution of your command line script is done by simply adding the
``renku run`` command before the actual command. ``renku run`` will detect:
```